### DISCUSSION TITLE
V5 rebooting constantly after intially working fine

### DISCUSSION THREAD

**Discussion by Seroczynski** [sanni/cartreader#700] [Need help]
Hi, build a couple of OCSR HW5 REV3 boards and working on getting them all put together. When testing one of them, I was able to dump several carts just fine (GBA > GB > GBC > MD > N64). After the N64 I was going to test SMS, and thus switching from 3V back to 5V, all of a sudden the cart reader started to reboot several times in a row, not stopping doing so until I power it down.
Now when I turn it on, it powers up fine, then after a couple of seconds it starts rebooting again. It's always first the LED that flickers before the screen turns off/on and it keeps repeating this. I've checked for the obvious shorts, but am wondering if this rings a bell to anyone on what might have happened or where I should look to in order to fix this.

**Edit:** I left it powered off for awhile and booted without a SD cart inserted. It worked fine, rebooted with SD cart and a SMS game and was able to dump it properly. I then moved on to dump a FC game, and it started rebooting again. ~Am I right in assuming this might have to do with the 470uF capacitor?~

**Edit 2:** The problem has to be on the top board, since it doesn't occur when I have that disconnected. I'll keep investigating a bit futher, but keep this thread open in the hope someone recognizes the issue I'm having.

**Comment by Cephiros** [comment]
Check for contacts that might be touching in the slots. I dont remember which but some of then have vcc and gnd direcrly across from one another. You can try putting a cart in every slot(without dumping) and see if it stops the rebooting. I would suspect the n64 slot personally, because that was the last slot used prior to the problem appearing after cart removal.
